pallets/manta-pay/src/lib.rs

```diff
@@ -270,6 +283,15 @@ pub mod pallet {
                     && post.sink_accounts.is_empty(),
                 Error::<T>::InvalidShape
             );
+            for post in post.receiver_posts.clone() {
```

Comment:
is it better to use iter() instead of clone()?

Reply:
indeed.